api/envoy/v7/http/service_control/config.proto

```diff
@@ -110,26 +110,22 @@ message FilterConfig {
   oneof access_token {
     option (validate.required) = true;
 
-    // The Instance Metadata Server uri used to fetch access token from Instance
+    // Uri used to fetch access token from Instance
     // Metadata Server.
     espv2.api.envoy.v7.http.common.HttpUri imds_token = 4;
 
-    // The local path or inline content of the service account json file used to
-    // generate access token.
-    espv2.api.envoy.v7.http.common.DataSource service_account_secret = 5;
-
     // Information used to fetch access token from Google Cloud IAM.
-    espv2.api.envoy.v7.http.common.IamTokenInfo iam_token = 6;
```

Comment:
Keep these field numbers as is. Just add `reserved 5` to the top of this message. Then we don't need a new config version, this is completely backwards compatible for API Gateway.